delivery/meteor-delivery.js

```diff
@@ -1,22 +1,21 @@
 import { Router } from 'express';
 import { executeApi } from '../repository/meteor-repository.js';
 import { getMeteors } from '../use_cases/meteor-use-cases.js';
+import CustomException from '../class/CustomException.js';
 
 const router = Router();
 
-router.get('/api/meteors', async (req, res) => {
+router.get('/meteors', async (req, res, next) => {
     try {
         const params = {
             date: req.query.date || new Date().toISOString().split('T')[0],
-            count: req.query.count || -1,
-            isDangerous: req.query['were-dangerous-meteors'] === 'true' || undefined
+            count: String(req.query.count).toLowerCase() === 'true',
+            isDangerous: req.query['were-dangerous-meteors']
```

Comment:
all types should be prepared for use cases inside of controller
isDangerous - is boolean? then need to provide correct type

Reply:
Fixed.